### PR TITLE
Update readme with how to remove default headers

### PR DESCRIPTION
**Pull request by Samic8**
## Description

Update readme with how to remove default headers with the current API. If this seems too hacky, I'm happy to look into implementing a new option to provide the same functionality.

### Documentation

This is only a documentation change

### CODE REVIEW

File: packages/gatsby-plugin-netlify/README.md [hunk 1]
```diff
@@ -95,6 +95,24 @@ have to add to every path. To make this easier, the plugin provides the
...
+#### Removing default headers
+
+This plugin provides default headers for increased security. If you need to remove any of those headers you can in the transfromHeaders option:
+
+```
+options: {
+    transformHeaders: (headers, path) => {
+        // Remove default security headers to allow cross-origin iframe
+        if (path === "/*") {
+            return _.pickBy(headers, (value) => {
```
Comment (Samic8) [inline packages/gatsby-plugin-netlify/README.md:107]: Using lodash in this example to keep this example conscience. Happy to replace with more verbose vanilla JS if needed.

**Comment by Samic8** [comment]
Are the failing tests related to this documentation change?

**Comment by pvdz** [comment]
> Are the failing tests related to this documentation change?

No, please ignore that. (Somebody else will eventually process this PR, may take some time)

File: packages/gatsby-plugin-netlify/README.md [hunk 2]
```diff
@@ -98,6 +98,24 @@ have to add to every path. To make this easier, the plugin provides the
...
+                return !value.includes("X-Frame-Options");
+            });
+        }
```
Comment (wardpeet) [inline packages/gatsby-plugin-netlify/README.md:113]: Let's switch to native syntax
```suggestion
        if (path === "/*") {
            return headers.filter(
              header => !header.toLowerCase().includes(`X-Frame-Options`)
            )
        }
```

**Review by wardpeet — COMMENTED** [review]
Thanks for the PR, I'm not sure if the example you're giving is the best one. As if you want to disable default security headers, you should use `mergeSecurityHeaders: false`.

**Comment by AishaBlake** [comment]
> Thanks for the PR, I'm not sure if the example you're giving is the best one. As if you want to disable default security headers, you should use `mergeSecurityHeaders: false`.

Yes, thank you! This option does seem to take care of the issue. If you're trying to demonstrate a more general point, @Samic8, would you mind either outlining your thought process in more detail in the PR description or opening an issue?

**Comment by LekoArts** [comment]
Thanks for the PR! I'll close this as commented by my colleague we already have a plugin option for that exact use case. Please open an issue if you want to explain your issue in more detail. Thanks!

**Pull request closed without merging**